exercises/series/src/series.c

```diff
@@ -0,0 +1 @@
+#include "example.c"
```

Comment:
Remove this extra file and everything looks good to me 👍 

[The x-common contributing guide](https://github.com/exercism/x-common/blob/master/CONTRIBUTING.md) has a decent section on using git for dealing with changes to master :)